src/network/netdev/wireguard.c

```diff
@@ -643,7 +655,7 @@ int config_parse_wireguard_preshared_key_file(
         if (path_simplify_and_warn(path, PATH_CHECK_ABSOLUTE, unit, filename, line, lvalue) < 0)
```

Comment:
We should probably start making use of `PATH_CHECK_NON_API_VFS` in networkd parsers too :)

Reply:
-> #34015.